station-vanilla-fix-v0/src/main/java/net/modificationstation/stationapi/impl/vanillafix/recipe/VanillaFuelItemFixImpl.java

```diff
@@ -22,7 +23,8 @@ private static void registerFuel(RecipeRegisterEvent event) {
                     FuelRegistry.addFuelItem(Item.ITEMS[block.id], 300);
             FuelRegistry.addFuelItem(Item.STICK, 100);
             FuelRegistry.addFuelTag(ItemTags.COALS, 1600);
-            FuelRegistry.addFuelItem(Item.LAVA_BUCKET, 20000);
+            FuelRegistry.addFuelItem(new ItemStack(Item.LAVA_BUCKET, 1), 20000);
+            FuelRegistry.addFuelItem(new ItemStack(Block.SLAB.asItem(), 1, 0), 100);
```

Comment:
A couple of things:
1. Why change the lava bucket line?
2. Wooden slab isn't usable as fuel in vanilla.